catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/TestDorisDatabaseOperations.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.doris.integration.test;
+
+import com.datastrato.gravitino.utils.RandomNameUtils;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("gravitino-docker-it")
+public class TestDorisDatabaseOperations extends TestDorisAbstractIT {
+
+  @Test
+  public void testBaseOperationDatabase() {
+    String databaseName = RandomNameUtils.genRandomName("ct_db");
+    Map<String, String> properties = new HashMap<>();
+    String databaseComment = "test_comment";
+
+    Assertions.assertThrows(
```

Comment:
Could you please also assert the exception message?

Reply:
[DorisDatabaseOperations](https://github.com/datastrato/gravitino/blob/main/catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisDatabaseOperations.java#L24)

`throw new UnsupportedOperationException();`

so, maybe it's no need to assert the exception message